src/OpenApi/Serializer/OpenApiNormalizer.php

```diff
@@ -95,4 +95,44 @@ public function hasCacheableSupportsMethod(): bool
 
         return true;
     }
+
+    private function getPathsCallBack(): \Closure
+    {
+        return static function ($decoratedObject): array {
+            if ($decoratedObject instanceof Paths) {
+                $paths = $decoratedObject->getPaths();
+
+                // sort paths by tags, then by path for each tag
+                uksort($paths, function ($keyA, $keyB) use ($paths) {
+                    $a = $paths[$keyA];
+                    $b = $paths[$keyB];
+
+                    $tagsA = [
+                        ...($a->getGet()?->getTags() ?? []),
+                        ...($a->getPost()?->getTags() ?? []),
+                        ...($a->getPut()?->getTags() ?? []),
```

Comment:
it should be `patch` instead as `put` will no longer be there by default in 4.0

Reply:
That's a mistake on my part, I intended to add both. I still leave Put in case it's used, don't I?

Reply:
It's added